presto-product-tests/src/main/java/com/facebook/presto/tests/jdbc/JdbcTests.java

```diff
@@ -72,20 +71,12 @@ public Requirement getRequirements(Configuration configuration)
         }
     }
 
-    private Connection connection;
-
-    @BeforeTestWithContext
-    public void setup()
-    {
-        connection = defaultQueryExecutor().getConnection();
-    }
-
     @Test(groups = JDBC)
     @Requires(ImmutableNationTable.class)
     public void shouldExecuteQuery()
             throws SQLException
     {
-        try (Statement statement = connection.createStatement()) {
+        try (Statement statement = connection().createStatement()) {
```

Comment:
why this is better?

Reply:
I am bit a reluctant of having fields in product test classes. These variables may leak and causes product tests to be not eligible to be run in parallel.